src/browser-polyfill.js

```diff
@@ -509,7 +509,7 @@ if (typeof browser === "undefined" || Object.getPrototypeOf(browser) !== Object.
 
   // The build process adds a UMD wrapper around this file, which makes the
   // `module` variable available.
-  module.exports = wrapAPIs(chrome); // eslint-disable-line no-undef
```

Comment:
Well, the&nbps;ESLint CommonJS environment only allows `require` and&nbsp;`module.exports`:

https://github.com/sindresorhus/globals/blob/06fb76415e4c1f664c8e0a1185f800956b59fd45/globals.json#L1029-L1034

Reply:
@ExE-Boss I know what the commonjs eslint environment is explicitly allowing, and that's exactly what I mean in https://github.com/mozilla/webextension-polyfill/pull/167#discussion_r246878903:

- I prefer that we prevent `require` to be used in this UMD module, because the UMD module is supposed to be self-contained (and this is also a reason why this line https://github.com/mozilla/webextension-polyfill/blob/master/src/browser-polyfill.js#L22 is not a `require` statement)

- I prefer to be sure that eslint is preventing `module.exports` to be used anywhere else, because the UMD module is supposed to export only that single property and I want to be sure that no other part of the polyfill ever tries to exports anything else.

Reply:
I agree with @rpl's assessment above. Currently there are three variables in the line, and the intention is to apply the var only to `module.exports`. If you really like to minimize the usage of the eslint rule, then you could use:

```javascript
module.exports = // eslint-disable-line no-undef
  wrapAPIs(chrome);
```

I think that leaving the code untouched (before this PR) is also fine though.

Reply:
I decided to&nbsp;declare <code>module:&nbsp;true</code> in&nbsp;the&nbsp;`.eslintrc` for&nbsp;the&nbsp;`src` directory.